common/src/main/scala/com/gu/media/upload/actions/UploadActionHandler.scala

```diff
@@ -17,6 +19,9 @@ abstract class UploadActionHandler(store: UploadsDataStore, plutoStore: PlutoDat
 
   private val s3 = uploaderAccess.s3Client
   private val bucket = uploaderAccess.userUploadBucket
+  private val transcoderClient = uploaderAccess.transcoderClient
+  transcoderClient.setRegion(Region.getRegion(Regions.EU_WEST_1))
+  private val TRANSCODER_PRESET_ID = "1351620000001-000001" //System preset: Generic 1080p
```

Comment:
Yeah likely - I was thinking of waiting till we see this end to end because it might be that we need to transcode it in a few formats so this might need to change to a list.